packages/config/config/devices/0x0234/zdb5400.json

```diff
@@ -0,0 +1,581 @@
+		},
+		"14[0xff0000]": {
+			"$import": "templates/logic_group_template.json#global_brightness",
+			"label": "Global brightness Green LED"
+		},
+		"14[0xff00]": {
+			"$import": "templates/logic_group_template.json#global_brightness",
+			"label": "Global brightness Blue LED"
+		},
+		"14[0xff]": {
+			"$import": "templates/logic_group_template.json#multilevel_disabled_byte"
+		},
+		"15[0x01]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 2 secure message",
+			"valueSize": 2
+		},
+		"15[0x02]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 3 secure message",
+			"valueSize": 2
+		},
+		"15[0x04]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 4 secure message",
+			"valueSize": 2
+		},
+		"15[0x08]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 5 secure message",
+			"valueSize": 2
+		},
+		"15[0x10]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 6 secure message",
+			"valueSize": 2
+		},
+		"15[0x20]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 7 secure message",
+			"valueSize": 2
+		},
+		"15[0x40]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 8 secure message",
+			"valueSize": 2
+		},
+		"15[0x80]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 9 secure message",
+			"valueSize": 2
+		},
+		"15[0x100]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 10 secure message",
+			"valueSize": 2
+		},
+		"15[0x0200]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 11 secure message",
+			"valueSize": 2
+		},
+		"15[0x0400]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 12 secure message",
+			"valueSize": 2
+		},
+		"15[0x0800]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 13 secure message",
+			"valueSize": 2
+		},
+		"15[0x1000]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 14 secure message",
+			"valueSize": 2
+		},
+		"15[0x1fff]": {
+			"label": "Messages in all association groups are sent as secure",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 8191,
+			"defaultValue": 8191,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 8191
+				}
+			]
+		},
```

Comment:
No overlaps please

Reply:
This can be removed from the code. references overlapping bit masks